django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,17 @@
+import enum
+
 from django.db import router
 
 
+class OperationCategory(enum.Enum):
+    ADDITION = "+"
+    REMOVAL = "-"
+    ALTER = "~"
+    PYTHON = "p"
+    SeparateDatabaseAndState = "?"
```

Comment:
Maybe use `MIXED` here? We should at least not mix camel and all caps case.

Reply:
Based on [this](https://mail.python.org/pipermail/python-ideas/2016-September/042340.html), I changed it to `UPPER_CASE`.